tests/entropy/entropy-02/test.yaml

```diff
@@ -0,0 +1,94 @@
+requires:
+  min-version: 8.0.1
+
+pcap: ../entropy-01/input.pcap
+
+args:
+  - --set logging.entropy.make-unique=on
+
+checks:
+  - filter:
+      count: 1
+      match:
+        event_type: alert
+        alert.signature_id: 1
+        metadata.entropy[0].__ignore_type: "{'sid:1;buffer:file_data;instance:1': 4.137370175000773}"
+        metadata.entropy[8].__ignore_type: "{'sid:10;buffer:file_data;instance:1': 4.150007324019584}"
```

Comment:
Can we just use the object here instead of `__ignore_type`, for example: https://github.com/OISF/suricata-verify/pull/2665/files#diff-01fcb3749573dcf5e88b90390ba73a6b8fe3b38f4677e2df07026e2f52ddb198